mito-ai/src/Extensions/AiChat/ChatHistoryManager.tsx

```diff
@@ -98,7 +98,7 @@ export class ChatHistoryManager {
         }
     }
 
-    updateMessageAtIndex(index: number, newContent: string): IOutgoingMessage {
+    updateMessageAtIndex(index: number, newContent: string, isAgentMessage: boolean = false): IOutgoingMessage {
```

Comment:
If I update the original message I send to the agent, I want it to rerun the agent:planning stuff. But currently, it seems to just send it as a regular chat message